src/App.js

```diff
@@ -242,11 +247,24 @@ class App extends Component {
 		paginatedItems = paginate(sortedItems, 10, currentPage)
 		console.log(paginatedItems)
 
-
+		let pagination = null
```

Comment:
here you can use conditional rendering:
<div class="...">
{
condition && <span>1</span>
}
{
condition2 && <span>2</span>
}
</div>

Reply:
Changed.